src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -189,9 +199,26 @@ public sealed class ProcessSystem : AEntitySetSystem<float>
 
                 if (processData.WritableOutputs.TryGetValue(ATP, out amount))
                 {
-                    processATPProduction += amount;
-
                     result.AddProduction(organelle.Definition.InternalName, amount);
+
+                    var isInPatch = true;
+
+                    foreach (var input in processData.WritableInputs)
+                    {
+                        if (biome.Compounds.TryGetValue(input.Key, out var inputCompoundData))
```

Comment:
Actually doesn't this not see any chunk type compounds? If that is the case then I guess it would make sense to re-introduce that separate variable tracking compound balance when relying purely on clouds. Sorry for not realizing this sooner.

Reply:
I think it might be worth actually checking for the compound in chunks if the first check fails. That will make it work as intended.

Reply:
It could but at that point I think the compound amount calculation method should be split out to a helper method as the editor also needs to display this information (it would make sense if process numbers and GUI display numbers were the same, just with some scaling probably in the editor display).

Reply:
Well the separated function exists now, but I'm not sure what to do with it.